src/reference/asciidoc/whats-new.adoc

```diff
@@ -179,6 +179,12 @@ Also the `ConnectionBlockedEvent` and `ConnectionUnblockedEvent` events are emit
 
 See <<connections>> for more information.
 
+===== Connection Factory Bean Changes
+
+The `RabbitConnectionFactoryBean` now provides an `enabaleHostnameVerification` property; set it to `true` to enable host name verification.
+Also `setHostnameVerifier` is provided for when using Java 6; see the connection factory javadocs for more information.
```

Comment:
There is no such a functionality in this version.
Also the commit message is misleading.

Will fix on merge...

Reply:
Oops - yes; thanks.